### PR TITLE
Fix slugs

### PR DESCRIPTION
**Pull request by bricelam**
🐌

image

### CODE REVIEW

File: entity-framework/core/miscellaneous/cli/dotnet.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Run the following commands to verify that EF Core CLI tools are correctly instal
 
 The output from the command identifies the version of the tools in use:
 
-```console
+```output
```
Comment (smitpatel) [inline entity-framework/core/miscellaneous/cli/dotnet.md:73]: consistency issue in output vs Output

File: entity-framework/core/providers/sqlite/spatial.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ This page includes additional information about using spatial data with the SQLi
 
 On Windows, the native mod_spatialite library is distributed as a NuGet package dependency. Other platforms need to install it separately. This is typically done using a software package manager. For example, you can use APT on Debian and Ubuntu; and Homebrew on MacOS.
 
-``` sh
+``` bash
```
Comment (smitpatel) [inline entity-framework/core/providers/sqlite/spatial.md:16]:
```suggestion
```bash
```

**Review by smitpatel — APPROVED** [review]

File: entity-framework/core/miscellaneous/cli/dotnet.md (hunk 1 above)
Comment (bricelam) [reply entity-framework/core/miscellaneous/cli/dotnet.md:73]: What's inconsistent? Console is for commands you can run in a console; output is for the output of console applications.

File: entity-framework/core/providers/sqlite/spatial.md (hunk 2 above)
Comment (bricelam) [reply entity-framework/core/providers/sqlite/spatial.md:16]: Sure, I'll normalize. I was surprised to see that the majority had a space

File: entity-framework/core/miscellaneous/cli/dotnet.md (hunk 1 above)
Comment (smitpatel) [reply entity-framework/core/miscellaneous/cli/dotnet.md:73]: first letter capital/non-capital in different places.

Comment (bricelam) [reply entity-framework/core/miscellaneous/cli/dotnet.md:73]: Ah, fixed.

**Pull request merged**